esp/scm/ldapenvironment.ecm

```diff
@@ -0,0 +1,102 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+ESPrequest LDAPQueryDefaultsRequest
+{
+};
+
+ESPresponse [exceptions_inline] LDAPQueryDefaultsResponse
+{
+    int     retcode;
+    string  HPCCRootName;
+    string  AdminGroupName;
+    string  sharedFilesBaseDN;
+    string  sharedGroupsBaseDN;
+    string  sharedUsersBaseDN;
+    string  sharedResourcesBaseDN;
+    string  sharedWorkunitsBaseDN;
+};
+
+/*##############################################################################*/
+
+ESPenum OUMode : int
+{
+    GeneratePrivate(0, "Generate private OU branch, using LDAP Root and Environment name (Recommended)"),
```

Comment:
I wonder if "stand alone" or "independent" would be a better term than "private"?  Private just sounds like it could mean something else security wise.

Reply:
Changed wording to "stand alone"